test/recipes/controls/aws_parallelcluster_slurm/install_slurm_spec.rb

```diff
@@ -0,0 +1,93 @@
+# Copyright:: 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+slurm_user = 'slurm'
+slurm_group = slurm_user
+slurm_license_path = '/opt/parallelcluster/licenses/slurm'
+
+control 'slurm_installed' do
```

Comment:
We may want to expand this:
- we should be able to find the most common binaries under `/opt/slurm/bin` (e.g. `srun`,`sbatch`, `scontrol`, etc.);
- we may search for specific libraries we expect to be there (for example `accounting_storage_mysql.so` to check that the mysql plugin has been compiled);
- later on we could think of more stuff.

Reply:
I only partially agree.
I do not think we should test for the presence of the binaries, we are building the code from the sources ...so this check would actually test the `slurm makefile` and install process ..that is outside our scope.

I however agree on testing for the presence of the expected libraries since this may depend on how we configure the environment while building slurm, so something may be missing if we have some bug or regression.